src/Microsoft.Health.Fhir.Tests.Common/TestFiles/Normative/Appointment.json

```diff
@@ -47,7 +47,6 @@
             }
         ]
     },
-    "priority": 5,
```

Comment:
>"priority": 5, [](http://example.com/codeflow?start=4&length=14)

as alternative you can add new file to R5 folder with priority been CodeableConcept


"priority":
 {
    "coding": [
      {
        "system": "[http://terminology.hl7.org/ValueSet/v3-ActPriority](http://terminology.hl7.org/ValueSet/v3-ActPriority)",
        "code": "EM",
        "display": "emergency"
      }
    ],
    "text": "Emergency"
  }